tests/apitest/src/ObjCRuntime/SimpleRegistrarTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using NUnit.Framework;
+
+#if !XAMCORE_2_0
+using MonoMac.Foundation;
+using MonoMac.ObjCRuntime;
+#else
+using Foundation;
+using ObjCRuntime;
+
+#endif
+
+namespace Xamarin.Mac.Tests
+{
+	[Register ("RegistrarTestClass")]
+	class RegistrarTestClass : NSObject
+	{
+		public virtual string Value
+		{
```

Comment:
I think you need to configure your editor to put curly braces in the right place :smile: